### ISSUE TITLE
Pre-train models

### ISSUE THREAD

**Issue by kequanpan** [littlebeen/Diffusion-Enhancement-for-CR#8]
I have already used your pre-train models on RICE2 but the output is not what i expected. Can you provide some models that are trained in CUHK datasets and divide the RICE datasets into train datasets and test datasets? Thanks

**Comment by littlebeen** [comment]
Sorry, I don't fully understand your question. Is the pre-trained model could not remove the cloud in RICE2? The separation of the train and test set of RICE2 is random. I could provide their image ids if you want.

**Comment by kequanpan** [comment]
Happy to receive your reply. I rethought the problems I encountered while studying your paper. 
1.I followed the instructions on GITHUB and successfully ran the code  based on the RICE2 dataset. But the outputs are a bit different. In mn2, my PSNR is 29.373, SSIM is 0.8825 , LPIPS is 0.0540.(Compared your outputs:33.637,0.8825,0.0476). In mdsa2, my PSNR is 29.872, SSIM is 0.8869, LPIPS is 0.0491.(Compared your outputs:33.598,0.8842,0.0452). I haven't found the cause of this problem yet.
2.In the usage, THE pretrain CR model Part, the models(.pth) are put tinto guided_diffusion/cloudnet/........respectivily. But in THE Test Part, the code need to put the pre-train model into File'Pre_train', i used the models you uploaded on RICE2 with mn and mdsa(The file suffix is ​​pt). if I want to change the dataset to CUHK, do I need to use the pre-trained model on CUHK? Currently, your page has only uploaded the pre-trained model for RICE2.
3.You mentioned pre-trained models in two places in your description, and I can't distinguish the two pre-trained models well.
4.Does each model have its pre-trained parameters? If not, how can I pre-train each model?
I am  very interested in the direction of your paper, and I am also in the learning stage. I would be very grateful if you could provide some help.

**Comment by littlebeen** [comment]
1. The difference between your result and my result may due to the difference of our test set.
2. I will upload the pre-train model as soon as possible. But I am quite busy recently. Welcome to push me.
3. Our pre-train model could be divided into two prat. The reference visual prior model (an end-to-end model trained in the other github project link: https://github.com/littlebeen/Cloud-removal-model-collection) and our diffusion model (trained in this project, please follow the train guidance).

**Comment by kequanpan** [comment]
Thank you for replyinge me. Recently I studied the method you proposed through the code you shared on GITHUB. And I learned a lot from your approach. At the same time, I am exploring this approach further. I still have a few questions that I haven't answered yet, I will sort them out by the end of this week and send them to you.
And I will try to train some end-to end models by myself to see if its results are different.

**Comment by littlebeen** [comment]
Add my wechat if you want sjl1506609605
